spec/models/book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe Book do
+  subject(:book) { create(:book) }
```

Comment:
you could use `build_stubbed` here in order to improve the performance ot this spec